src/main/java/pages/HomePage.java

```diff
@@ -21,4 +45,59 @@ public CatalogPage clickOnCatalogsPageButton() {
         logger.debug("Clicked successfully on the catalog page button");
         return new CatalogPage();
     }
+
+    @Step("Click on the services page button")
+    public ServicesPage clickOnTheServicesPageButton() {
+        servicesButton.click();
+        logger.debug("Clicked successfully on the services page button");
+        return new ServicesPage();
+    }
+
+    @Step("Click on the logotype")
+    public HomePage clickOnTheLogotype() {
+        logo.click();
+        logger.debug("Clicked successfully on the logo");
+        return new HomePage();
+    }
+
+    @Step("Input text in the search field")
+    public HomePage searchQueryInput() {
+        searchField.sendKeys(SEARCH_QUERY);
```

Comment:
Why search value is hardcoded inside a method via a constant? if you want to search with a different query you will create a new method?